robottelo/ui/oscappolicy.py

```diff
@@ -27,6 +27,8 @@ def create(self, name, desc=None, content=None, profile=None, period=None,
         if desc:
             self.text_field_update(locators['oscap.desc_policy'], desc)
         self.click(common_locators['submit'])
+        if self.wait_until_element(common_locators['haserror']):
+            return
```

Comment:
This is added to handle `neagtive_create` scenario.  This page cannot be submitted in ONE GO and hence this check.